tailnet/test/integration/network.go

```diff
@@ -66,9 +66,11 @@ func (c TestNetworkingClient) ResolveDERPMap() (*tailcfg.DERPMap, error) {
 	return basicDERPMap(c.ServerAccessURL)
 }
 
-type TestNetworkingProcess struct {
-	// NetNS to enter. If nil, the current network namespace is used.
-	NetNS *os.File
+type NetNamespace struct {
```

Comment:
This was called Process because the idea was we'd add other fields that would be passed to the process in the future, such as environment variables

If you want to move the NetNS into it's own type that sounds good to me, but I think it should still be wrapped in a process type for future extensibility.

Reply:
What "process" are you talking about?  As far as I can tell it's just a network namespace

Reply:
These values are associated to each subprocess, the server, client1 and client2. If you're going to change it I would prefer something like below instead so we can add other fields to each process in the future:

```go
type TestProcess struct {
  NetNS NetNamespace
}

type NetNamespace struct {}
```

Reply:
What's the difference between a "TestProcess" and "TestNetworkingClient" / "TestNetworkingServer"?

I'm still hung up on the fact that when `TestNetworking` is created/returned, there are no subprocesses running, so how can we be returning a "process"?  Are you planning for it to be some kind of run configuration for when we later start the process?  Can you give me a hypothetical example of a field that would be in `TestProcess`?

Reply:
Sorry; clearly didn't read your initial response closely, since you gave environment variables as an example.